### ISSUE TITLE
global-css breaking?

### ISSUE THREAD

**Issue by justinbmeyer** [stealjs/steal-tools#478]
This guide: https://gist.github.com/justinbmeyer/9b2b300dc300b9b29e5d90ec8d00b3ff

With this script

```js
var stealTools = require("steal-tools");
stealTools.export({
  system: {
    main: "myhub/repos/repos",
    config: __dirname+"/package.json!npm"
  },
  options: {
    verbose: true
  },
  outputs: {
    "+global-css": {}
  }
});
```

breaks with:

```
OUTPUT: +global-css
Cleaning...
Transpiling...
Potentially unhandled rejection [1] TypeError: Cannot read property 'load' of undefined
    at transform (/Users/justin/dev/myhub/node_modules/steal-tools/lib/build/transform.js:119:32)
    at transformAndWriteOut (/Users/justin/dev/myhub/node_modules/steal-tools/lib/build/export.js:130:18)
    at doTransform (/Users/justin/dev/myhub/node_modules/steal-tools/lib/build/export.js:196:6)
    at /Users/justin/dev/myhub/node_modules/steal-tools/lib/build/export.js:208:6
    at tryCatchReject (/Users/justin/dev/myhub/node_modules/steal-es6-module-loader/dist/es6-module-loader.src.js:1183:30)
    at runContinuation1 (/Users/justin/dev/myhub/node_modules/steal-es6-module-loader/dist/es6-module-loader.src.js:1142:4)
    at Fulfilled.when (/Users/justin/dev/myhub/node_modules/steal-es6-module-loader/dist/es6-module-loader.src.js:930:4)
    at ContinuationTask.run (/Users/justin/dev/myhub/node_modules/steal-es6-module-loader/dist/es6-module-loader.src.js:1040:24)
    at Scheduler._drain (/Users/justin/dev/myhub/node_modules/steal-es6-module-loader/dist/es6-module-loader.src.js:97:19)
    at Scheduler.drain (/Users/justin/dev/myhub/node_modules/steal-es6-module-loader/dist/es6-module-loader.src.js:62:9)
```

my guess is that this is b/c there is no css to export.  We should still probably handle this case.

**Comment by justinbmeyer** [comment]
Here's a zip file of my project [myhub.zip](https://github.com/stealjs/steal-tools/files/331118/myhub.zip)

**Comment by matthewp** [comment]
This should be fixed in 0.16.5.
